core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -756,6 +756,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey UNDERFS_S3A_DEFAULT_MODE =
+      new Builder(Name.UNDERFS_S3A_DEFAULT_MODE)
+          .setDefaultValue("0700")
```

Comment:
What format is this called? octal? because we should mention that in the description.

Reply:
yes, its octal